src/pyOpenMS/pxds/MRMFeatureQC.pxd

```diff
@@ -58,5 +66,5 @@ cdef extern from "<OpenMS/ANALYSIS/OPENSWATH/MRMFeatureQC.h>" namespace "OpenMS"
         
         libcpp_vector[MRMFQC_ComponentQCs] component_qcs
         libcpp_vector[MRMFQC_ComponentGroupQCs] component_group_qcs
-        libcpp_vector[MRMFQC_ComponentGroupQCs] component_group_pair_qcs
+        libcpp_vector[MRMFQC_ComponentGroupPairQCs] component_group_pair_qcs
```

Comment:
I dont think `ComponentGroupPairQCs` is defined anywhere, did this compile on your machine?